controlsfx/src/main/java/org/controlsfx/validation/ValidationResult.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2014, ControlsFX
+ * Copyright (c) 2014, 2019, ControlsFX

```

Comment:
Can you remove the last comma? As explained [here](https://github.com/controlsfx/controlsfx/wiki/Contributing-to-ControlsFX), the general format is :

> If you are editing an existing source file, the format should be , . For example, if the file was created in 2012, and it is currently 2014, it should say '2012, 2014'.

The rest seems good 👍